zebrad/tests/acceptance.rs

```diff
@@ -40,13 +41,15 @@
 //!
 //! $ cargo test sync_large_checkpoints_mempool_mainnet -- --ignored --nocapture
 //!
-//! $ sudo mkdir /zebrad-cache
-//! $ sudo chmod 777 /zebrad-cache
+//! $ export ZEBRA_CACHED_STATE_DIR="/zebrad-cache"
+//! $ sudo mkdir -p "$ZEBRA_CACHED_STATE_DIR"
```

Comment:
if we use `/var/cache/` instead, then being root is maybe not needed ?

Reply:
It will depend on the host the user is running. Most of the time is not needed, that's for certain, but it might be required in more security-tight environments, or if they decide to keep the default `/zebrad-cache`, which I didn't change in the comment to keep backwards compatibility

Reply:
Ok, thanks.